lib/rules/forbid-component-props.js

```diff
@@ -38,6 +45,9 @@ module.exports = {
           defaultAllowOnDOM: {
             type: 'boolean'
           },
+          defaultAllowOnComponents: {
+            type: 'boolean'
+          },
           forbid: {
```

Comment:
Hm, yeah i guess if i wanted to override a `forbid` i'd now just modify the object inside the array. This is fine as-is.